modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -856,4 +919,21 @@ FrontendDataSet.defaultProps = {
 	style: 'default',
 };
 
+function loadClientExtensions(importsHandlers) {
```

Comment:
Could we put this in `utils/loadClientExtensions.ts` and apply TS?

Reply:
Yes. And I also want to remove `getComponentByModuleURL` since I don't think it is really necessary.